frontend/src/views/user/UserManagementView.vue

```diff
@@ -251,7 +252,8 @@ onMounted(async () => {
     await userService.searchUsers({
       active: true,
       idp: [IdentityProvider.IDIR],
-      includeUserGroups: true
+      includeUserGroups: true,
+      group: GROUP_NAME_LIST
```

Comment:
This will show IDIR users that are proponents. Unsure if that is desired functionality or if this should be limited to nav, supervisor, & admin groups. Please confirm with UX.

Reply:
changed to not have proponents